schema/encoding/jsonschema/benchmark_test.go

```diff
@@ -38,15 +39,25 @@ func BenchmarkEncoder(b *testing.B) {
 			},
 		},
 		{
-			Name:   `Schema=Student`,
+			Name:   `Schema=Simple`,
 			Schema: studentSchema,
 		},
+		{
+			Name:   `Schema=Complex1`,
+			Schema: requestSchema,
+		},
+		{
+			Name:   `Schema=Complex2`,
+			Schema: responseSchema,
+		},
 	}
+	buf := bytes.NewBuffer(make([]byte, 65535))
```

Comment:
Actually, we could also just run the encoding once before the sub-benchmark loop starts! That would also ensure we always have enough space!